EasyPost.Tests.Net/UserTest.cs

```diff
@@ -8,72 +7,146 @@ namespace EasyPost.Tests.Net
     public class UserTest
     {
         [TestInitialize]
-        public void Initialize() => ClientManager.SetCurrent("GxhY479LTioDWsGcEtSAfQ");
+        public void Initialize()
+        {
+            VCR.SetUp(VCRApiKey.Production, "user", true);
+        }
+
+        private static User RetrieveMe()
+        {
+            return User.RetrieveMe();
+        }
+
+        private static User CreateUser()
+        {
+            return User.Create(new Dictionary<string, object>
+            {
+                {
+                    "name", "Test User"
+                }
+            });
+        }
+
+        // This endpoint returns the child user keys in plain text, do not run this test.
+        [Ignore]
+        [TestMethod]
+        public void TestCreate()
```

Comment:
Please do what you need to delete child accounts after they've been created/tested - this will ensure a year from now we don't pollute this test account with hundreds of dummy child users.

Reply:
Despite this?
`// Due to our inability to create child users securely, we must also skip deleting them as we cannot replace the deleted ones easily.`

Reply:
Ah sorry, read too much and brain broke. No we aren't even creating a user so no need to delete.